jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/skaffold/SkaffoldConfiguration.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.maven.skaffold;
+
+import java.io.File;
+import java.util.Collections;
+import java.util.List;
+import org.apache.maven.plugins.annotations.Parameter;
+
+/** Skaffold specific Jib plugin configuration options. */
+public class SkaffoldConfiguration {
+
+  /** Skaffold specific Jib plugin configuration for files to watch. */
+  public static class Watch {
+    @Parameter List<File> buildIncludes = Collections.emptyList();
+    @Parameter List<File> includes = Collections.emptyList();
+    @Parameter List<File> excludes = Collections.emptyList();
+  }
+
+  /** Skaffold specific Jib plugin configuration for files to sync. */
+  public static class Sync {
+    @Parameter List<File> excludes = Collections.emptyList();
+  }
+
+  @Parameter Watch watch = new Watch();
```

Comment:
I think we can delete this field. It's not being used, since we parse this explicitly in `FilesMojoV2`.

Reply:
I went back and forth on that, I think this helps define the structure. Even though we're parsing it manually in Xpp3Dom parser, and never delegate to maven to populate this.

Reply:
I really wish we could just do this using our own annotation processor at compile time though.

Reply:
Yeah, leaving this is OK.

I remember the reason that we parse this manually in `FilesMojoV2` is about running a goal only at root vs. in every sub-module?

Reply:
Correct, `aggregator =  true` Mojos cannot populate a parameter for a single module.